.gitpod.yml

```diff
@@ -0,0 +1,31 @@
+tasks:
+  - name: Stargate
+    command: docker-compose up stargate
+  - name: api
+    command: |
+      sleep 35
+      docker-compose up eddiehub-api
```

Comment:
does this need `eddiehub-api`? I usually use `docker-compose` by itself

Reply:
I start api and stargate independent of each other, thats why I compose them scoped and not with `docker-compose up`.

Reply:
ah ok 👍